scalardb/test/scalardb/elle_append_2pc_test.clj

```diff
@@ -69,11 +69,11 @@
   (reify
     TwoPhaseCommitTransaction
     (getId [_] "dummy-id")
-    (^Optional get [_ ^Get _] (throw (CrudException. "get failed")))
-    (^void put [_ ^Put _] (throw (CrudException. "put failed")))
-    (^void prepare [_] (throw (PreparationException. "preparation failed")))
-    (^void validate [_] (throw (ValidationException. "validation failed")))
-    (^void commit [_] (throw (CommitException. "commit failed")))
+    (^Optional get [_ ^Get _] (throw (CrudException. "get failed" nil)))
```

Comment:
We need to make this change because the constructors of these exceptions have been changed.